test/library/draft/arrow/ArrowPrinting.chpl

```diff
@@ -0,0 +1,41 @@
+module ArrowPrinting {
+  require "ArrayDecl.chpl", "ArrowRecordDecl.chpl", "ParquetDecl.chpl";
+  private use ArrayDecl, ArrowRecordDecl, ParquetDecl;
+  use SysCTypes, CPtr;
+
+  // Functions for printing
+
+  proc printArray(array: GArrowArray) {
+    var error: GErrorPtr;
+    var str: c_string = garrow_array_to_string(array, c_ptrTo(error));
+    if(str == ""){
+      g_print("Failed to print: %s\n", error.deref().message);
+      g_error_free(error);
+      return;
+    }
```

Comment:
Could be boolean function, called like `if hasError(str, other needed stuff...) then return else g_print("%s\n",str);` and could be used in all 3 spots

Reply:
I think we need to discuss error handling in greater detail before implementing this solution. Not that this is a bad idea, just that the topic is a pretty big decision and I want to invite more discussion towards it.